StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+FormSpec.swift

```diff
@@ -78,8 +153,88 @@ extension PaymentSheetFormFactory {
             return makeIban(apiPath: spec.apiPath?["v1"])
         case .sepa_mandate:
             return makeSepaMandate()
+        case .placeholder(let spec):
+            return makePlaceholder(for: spec)
         case .unknown:
             return nil
         }
     }
+
+    func makePlaceholder(for spec: FormSpec.PlaceholderSpec) -> Element? {
+        let field = spec.field
+        guard field != .unknown else { return nil }
```

Comment:
Should we have any metrics for how often we're getting unknown fields? Feels like we shouldn't just be swallowing this

Reply:
I plan on adding analytics in another PR, I can include this if it makes sense